packages/p2p/src/components/modal-manager/modals/filter-modal/filter-modal.jsx

```diff
@@ -1,19 +1,69 @@
 import * as React from 'react';
 import { observer } from 'mobx-react-lite';
-import { Button, Checkbox, Icon, Loading, Modal, Text, ThemedScrollbars, ToggleSwitch } from '@deriv/components';
+import {
+    Button,
+    Checkbox,
+    DesktopWrapper,
+    Icon,
+    Loading,
+    MobileFullPageModal,
+    MobileWrapper,
+    Modal,
+    Text,
+    ThemedScrollbars,
+    ToggleSwitch,
+} from '@deriv/components';
 import { localize, Localize } from 'Components/i18next';
 import { useStores } from 'Stores';
 import FilterModalHeader from './filter-modal-header.jsx';
 import FilterModalSearch from './filter-modal-search.jsx';
 import FilterModalNoResults from './filter-modal-no-results.jsx';
 import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+import { isMobile } from '@deriv/shared';
+
+const FilterModalResult = observer(({ onChange, selected_methods }) => {
```

Comment:
I moved this component `FilterModalResult` out of the parent component `FilterModal` because there is a bug where due to the parent component re-initializing `FilterModalResult`, the checkboxes would have to be clicked twice in order to uncheck/check it